cmd/dms-uploader/dms-uploader.go

```diff
@@ -17,8 +17,9 @@ import (
 
 func run() int {
 	// cmd line args
-	uri := flag.String("uri", "", "Object storage URI with credentials.")
+	uri := flag.String("uri", "", "Object store URI with credentials.")
 	path := flag.String("path", "", "Destination path")
+	timeout := flag.Int("timeout", 10, "Upload timeout in seconds")
```

Comment:
**nit:** Could also use a `flag` of type [Duration](https://pkg.go.dev/flag#Duration), which can then be specified with a unit on the command line like `10s` or `2m`.